torch/quantization/fx/quantize.py

```diff
@@ -485,7 +493,9 @@ def is_observed(input_arg):
                     env[node.name] = observed_graph.create_node('call_module', observer_name, (load_arg(node),), {})
                     observed_node_names_set.add(node.name)
 
-        observed_graph.output(load_arg(model.graph.result))
+        if output_node is None:
+            raise RuntimeError('Graph had no output node!')
+        observed_graph.output(load_arg(output_node.args[0]))
```

Comment:
Why not just put this in the loop?

Reply:
The output value is actually also used at line 502. Not sure why this seemed to pass when I ran this in FBCode...